generator/internal/language/rust.go

```diff
@@ -633,10 +635,17 @@ func (*RustCodec) ToCamel(symbol string) string {
 // difficult examples.
 //
 // [spec]: https://spec.commonmark.org/0.13/#block-quotes
-func (*RustCodec) FormatDocComments(documentation string) []string {
+func (c *RustCodec) FormatDocComments(documentation string, state *api.APIState) []string {
 	inBlockquote := false
 	blockquotePrefix := ""
+	linkRegex, err := regexp.Compile(`\]\[([a-z_]+\.[a-zA-Z0-9_\.]+)\]`)
```

Comment:
nit: make this private global and use regexp.MustCompile. It would be good to leave a nice comment above this line too as I am personally awful at groking regexps.

Reply:
Done.